sechub-pds-solutions/phan/README.adoc

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+
+:toc:
+:numbered:
+
+= Phan + PDS
+
+Phan is a static analyzer for PHP that prefers to minimize false-positives. Phan attempts to prove incorrectness rather than correctness.
+In addition, the combination of Phan and PDS allows both to run in a Kubernetes cluster or on virtual machines.
+
+This folder contains the scripts necessary to run Phan Wrapper + PDS locally within a container. It also contains scripts to create the PDS + Phan container and push it to your container registry, as well as a helm chart to install and run Phan Wrapper + PDS in a Kubernetes cluster.
+
+== Current limitations
+
+- only docker images uploaded as `.tar` files can be scanned
+
+== Run Locally
+
+This is the easiest way to get started.
+
+=== Requirements
+
+Docker and Docker-Compose need to be installed:
+
+. https://docs.docker.com/engine/install/[Install Docker]
+
+. Linux: https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user[Use Docker as non Root user]
+
+. https://docs.docker.com/compose/install/[Install Docker-Compose]
+
+=== Single Instance
+
+Start a single instance by using the Bash wrapper script which does setup everything automatically for you.
+
+==== Scan script
+
+The folder contains a start script which does the manual steps for you:
+
+----
+./01-start-single-docker-compose.sh
+----
+
+==== Together with SecHub
```

Comment:
Please remove this section as it is not accurate.

Reply:
Done.